js/components/ourTeam/isValidTeam.js

```diff
@@ -0,0 +1,71 @@
+function isValidTeam(team){
+    let errors = [];
+
+    if(typeof team.active !== 'boolean'){
+        errors.push('ERROR: paslaugos aktyvumo statusas turi buti boolean.');
+    }
+
+    if(typeof team.profilePicture !== 'string'){
+        errors.push('ERROR: profilio nuotraukos pavadinimas turi buti tekstinis.')
+    } else {
+        if(team.profilePicture === ''){
+            errors.push('ERROR: profilio nuotraukos pavadinimas negali buti tuscias.')
```

Comment:
Trūksta validacijos, pavyzdžiui, jeigu nėra nuotraukos, gal net neverta rodyti to elemento (kas ir vyksta), nes tuomet atsiranda tuščias laukas, bet tuomet gal gali ateiti kitas 'active' elementas?

Reply:
Bus išspręsta kai bus generuojama su carousel.